ROMFS/px4fmu_common/init.d/rc.gnd_apps

```diff
@@ -0,0 +1,24 @@
+#!nsh
+#
+# Standard apps for fixed wing
+#
+
+#
+# Start the attitude and position estimator
+#
+ekf2 start
```

Comment:
Great! Let's then refactor this PR first, rebase it on master, and then look forward to that. It will be good to the community to have access to different estimators so to check which one is more suitable for their needs.